src/webassembly/instance.rs

```diff
@@ -329,10 +335,17 @@ impl Instance {
                         RelocationType::LibCall(LibCall::NearestF64) => {
                             math_intrinsics::nearbyintf64 as isize
                         },
-                        _ => unimplemented!()
-                        // RelocationType::Intrinsic(name) => {
-                        //     get_abi_intrinsic(name)?
-                        // },
+                        RelocationType::LibCall(LibCall::Probestack) => {
+                            __rust_probestack as isize
```

Comment:
Is this the right thing to do for probestack?

Reply:
Yup. This is inspired by wasmtime approach

Reply:
Okay!